dev/com.ibm.ws.jaxb_fat/fat/src/com/ibm/ws/jaxb/fat/LibertyJAXBTest.java

```diff
@@ -35,6 +38,10 @@
     @BeforeClass
     public static void setUp() throws Exception {
         ShrinkHelper.defaultDropinApp(server, APP_NAME, "jaxb.web");
+        if (JakartaEE9Action.isActive()) {
+            server.changeFeatures(Arrays.asList("jaxb-3.0", "servlet-5.0", "componenttest-2.0"));
```

Comment:
I thought this happened automatically with the EE9Actiion.

Reply:
The EE9_FEATURES_ARRAY array in JakartaEE9Action doesn't have jaxb-3.0 listed in it.  I am thinking that needs to be updated for this PR.